opengever/base/visible_users_and_groups_filter.py

```diff
@@ -0,0 +1,89 @@
+from opengever.ogds.models.service import ogds_service
+from opengever.workspace import is_workspace_feature_enabled
+from plone import api
+
+
+class VisibleUsersAndGroupsFilter:
+    """This singleton object provides functions to check if a
```

Comment:
```suggestion
    """Provides functions to check if a
```

The singleton part is not relevant here nor does this class implement the singleton pattern.